spec/features/teams_spec.rb

```diff
@@ -35,4 +35,53 @@
       end
     end
   end
+
+  context 'when current user is a team admin or root' do
```

Comment:
I think it would be more semantic to use `when current user is the current team admin or the root`.